rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -267,9 +267,9 @@ protected Void doInBackground(Void... voids) {
             boolean isUserAuthorized = syncUtils.verifyAuthorization();
 
             Activity currentActivity = RDTApplication.getInstance().getCurrentActivity();
-            boolean isFormActivity = currentActivity instanceof RDTJsonFormActivity;
+            boolean isFormSourceActivity = currentActivity instanceof PatientProfileActivity;
```

Comment:
Change this to `boolean isRegisterActivity = currentActivity instanceof CovidJsonFormActivity;`

Reply:
fixed.